lncfg/db.go

```diff
@@ -38,7 +38,7 @@ const (
 	SqliteBackend              = "sqlite"
 	DefaultBatchCommitInterval = 500 * time.Millisecond
 
-	defaultPostgresMaxConnections = 50
+	defaultPostgresMaxConnections = 20
```

Comment:
With the other fixes later in this commit, if we revert this (back to 50) do things still pass? 

If not, then we may want to implement clamping for an upper limit here.